contrib/dockerd-rootless-setuptool.sh

```diff
@@ -338,9 +341,12 @@ install_systemd() {
 		DOCKER_HOST="unix://$XDG_RUNTIME_DIR/docker.sock" $BIN/docker version
 		systemctl --user enable "${SYSTEMD_UNIT}"
 	)
+	# fix usernames containing backslash like DOMAIN\user
+	# that will clash with display below
+	username=$(id -un | sed 's/\\/\\\\/g')
```

Comment:
Can we make this a function to avoid code clone?

Reply:
Good idea to avoid duplicate code. I will add a USERNAME global variable.